domain/src/main/java/org/oppia/android/domain/oppialogger/OppiaLogger.kt

```diff
@@ -114,4 +118,64 @@ class OppiaLogger @Inject constructor(
       )
       .build()
   }
+
+  /**Logs non-fatal exception. See [ExceptionsController.logNonFatalException] for more context.*/
```

Comment:
I think it'd be better if we move the context creation related functions towards the end of this file.